tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true
```

Comment:
If you agree on creating `src` folder, you can add this line here:
```
"baseUrl": "./src"
```

Reply:
Added new `baseUrl` https://github.com/Lahphim/react-surveyor-challenge/pull/44/commits/f7bed671df20dc7da86f66821803a5d2dcbd0084